packages/integration-karma/test/light-dom/basic/index.spec.js

```diff
@@ -0,0 +1,24 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+
+import Test from 'x/test';
+describe('Basic Light DOM', () => {
+    it('should render properly', () => {
+        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', true);
+        const elm = createElement('x-test', { is: Test });
+        document.body.appendChild(elm);
+
+        expect(elm.shadowRoot).toBeNull();
```

Comment:
Create 2 tests for this, one checking if the `shadowRoot` is `null` and another one making sure the component is rendered in the light DOM.